js/views/pauseScreen.js

```diff
@@ -53,19 +53,22 @@ var PauseScreen = React.createClass({
   },
 
   handlePlayerMouseDown: function(e) {
+    e.persist();
     this.props.controller.state.accessibilityControlsEnabled = true;
     this.props.controller.state.isClickedOutside = false;
     this.props.handleVrPlayerMouseDown(e);
   },
   handlePlayerMouseMove: function(e) {
     e.preventDefault();
+    e.persist();
```

Comment:
Please make sure that these `preventDefault`'s, etc. are not applied to all videos if they are only needed for VR360. Not sure if this is the case here, but we should check.